From 552cb5dc9e9dffc7ee229bbda0e21f9dfae45fd2 Mon Sep 17 00:00:00 2001
From: David <Davidteri91@gmail.com>
Date: Tue, 22 Aug 2017 19:54:56 +0200
Subject: [PATCH] bt: use TARGET_BOARD_AUTO to override qcom hals

BOARD_IS_AUTOMOTIVE := true includes car packages on not car devices
see https://android.googlesource.com/platform/packages/services/Car/+/master/car-support-lib/Android.mk#50

Change-Id: I28898df1e8855347129039b5cb0d43975d3a5415
Signed-off-by: David Viteri <davidteri91@gmail.com>
---
 Android.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Android.mk b/Android.mk
index 60d373e..4f943f2 100644
--- a/Android.mk
+++ b/Android.mk
@@ -3,7 +3,7 @@ ifeq ($(call my-dir),$(call project-path-for,bt-vendor))
 LOCAL_PATH := $(call my-dir)
 
 # TODO:  Find a better way to separate build configs for ADP vs non-ADP devices
-ifneq ($(BOARD_IS_AUTOMOTIVE),true)
+ifneq ($(TARGET_BOARD_AUTO),true)
   ifneq ($(filter msm8x27 msm8974 msm8226,$(TARGET_BOARD_PLATFORM)),)
     include $(call all-named-subdir-makefiles,msm8960)
   else ifneq ($(filter msm8994,$(TARGET_BOARD_PLATFORM)),)
-- 
2.17.1

